astro/src/content/docs/get-started/core-concepts/applications.mdx

```diff
@@ -46,6 +46,29 @@ Below is a visual reminder of the relationships between FusionAuth's primary cor
 
 <img src="/img/docs/get-started/core-concepts/core-concepts-relationships-applications.png" alt="Diagram showing Applications used within FusionAuth" />
 
+## Universal Applications
+A universal application is an application that exists for all tenants, rather than belonging to a single tenant. Universal applications are useful when you need to make the same application available to users across many or all tenants.
+
+A universal application uses the same application configuration across all tenants, including its client Id, client credentials, and more. This makes configuration management simpler, allowing you to share configuration across all uses of the application.
+
+### Universal Application Registrations
+Users from any tenant can have a registration to a universal application, and registrations work as they would for normal applications.
+
+### Universal Applications and Tenants
+Becuase a universal application does not belong to a single tenant, you will generally have to supply a tenant Id when interacting with a universal application. This is true during authentication and other user-specific workflows.  The supplied tenant Id tells FusionAuth which tenant to look a user up in, to retrieve theme information from, and more.
```

Comment:
`which tenant to look a user up in` maybe `which tenant the user that is logging in belongs to?`